app/src/main/kotlin/com/wire/android/feature/e2ei/OAuthUseCase.kt

```diff
@@ -47,12 +47,9 @@ import java.security.SecureRandom
 class OAuthUseCase(
     context: Context,
     private val authUrl: String,
-    private val claims: JsonObject,
-    oAuthState: String?
+    private val claims: JsonObject
 ) {
-    private var authState: AuthState = oAuthState?.let {
-        AuthState.jsonDeserialize(it)
-    } ?: AuthState()
+    private var authState: AuthState = AuthState()
```

Comment:
I suggest to pass it as an argument, so we can pass a mock object when testing the use case.